specs/lib/commentOnClosedIssueSpec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var fsExtra = require('fs-extra');
+var Promise = require('bluebird');
+var requestPromise = require('request-promise');
+
+var getUniqueMatch = require('../../lib/getUniqueMatch');
+
+var commentOnClosedIssue = require('../../lib/commentOnClosedIssue');
+
+describe('commentOnClosedIssue', function() {
+    it('throws if `url` or `message` is undefined', function () {
+        expect(function() {
+            commentOnClosedIssue();
+        }).toThrowError();
+
+        expect(function() {
+            commentOnClosedIssue({});
+        }).toThrowError();
+    });
+
+    it('passes correct commentsUrl to _implementation', function() {
+        spyOn(commentOnClosedIssue, '_implementation').and.callFake(function(commentsUrl) {
```

Comment:
Rather than using `callFake` you can just use `expect(commentOnClosedIssue._implementation).toHaveBeenCalledWith('https://api.github.com/repos/baxterthehacker/public-repo/issues/2/comments');` below after you execute the function.